File: sprint5/By languages/Java/lesson8code1.java
class Node {
    int key;
    int height;
    Node left;
    Node right;

    public Node(int k, int h, Node l, Node r) {
        key = k;
        height = h;
        left = l;
        right = r;
    }

    public Node(int k) {
        this(k, 1, null, null);
    }
}

File: sprint5/By languages/swift/lesson8code1.swift
class Node<T: Comparable> {
    var key: T
    var left: Node<T>?
    var right: Node<T>?
    var height: Int
    
    init(key: T, height: Int = 1, left: Node<T>? = nil, right: Node<T>? = nil) {
        self.key = key
        self.left = left
        self.right = right
        self.height = height
    }
}